packages/flutter/lib/src/material/color_scheme.dart

```diff
@@ -4,167 +4,508 @@
+    Color? tertiaryContainer,
+    Color? onTertiaryContainer,
+    required this.error,
     required this.onError,
-    required this.brightness,
-  }) : assert(primary != null),
-       assert(primaryVariant != null),
-       assert(secondary != null),
-       assert(secondaryVariant != null),
-       assert(surface != null),
-       assert(background != null),
-       assert(error != null),
+    Color? errorContainer,
+    Color? onErrorContainer,
+    required this.background,
+    required this.onBackground,
+    required this.surface,
+    required this.onSurface,
+    Color? surfaceVariant,
+    Color? onSurfaceVariant,
+    Color? outline,
+    Color? shadow,
+    Color? inverseSurface,
+    Color? onInverseSurface,
+    Color? inversePrimary,
+    @Deprecated(
+      'Use primary or primaryContainer instead. '
+      'This feature was deprecated after v2.6.0-0.0.pre.'
+    )
+    Color? primaryVariant,
+    @Deprecated(
+      'Use secondary or secondaryContainer instead. '
+      'This feature was deprecated after v2.6.0-0.0.pre.'
+    )
+    Color? secondaryVariant,
+  }) : assert(brightness != null),
+       assert(primary != null),
        assert(onPrimary != null),
+       assert(secondary != null),
        assert(onSecondary != null),
-       assert(onSurface != null),
-       assert(onBackground != null),
+       assert(error != null),
        assert(onError != null),
-       assert(brightness != null);
+       assert(background != null),
+       assert(onBackground != null),
+       assert(surface != null),
+       assert(onSurface != null),
+       _primaryContainer = primaryContainer,
+       _onPrimaryContainer = onPrimaryContainer,
+       _secondaryContainer = secondaryContainer,
+       _onSecondaryContainer = onSecondaryContainer,
+       _tertiary = tertiary,
+       _onTertiary = onTertiary,
+       _tertiaryContainer = tertiaryContainer,
+       _onTertiaryContainer = onTertiaryContainer,
+       _errorContainer = errorContainer,
+       _onErrorContainer = onErrorContainer,
+       _surfaceVariant = surfaceVariant,
+       _onSurfaceVariant = onSurfaceVariant,
+       _outline = outline,
+       _shadow = shadow,
+       _inverseSurface = inverseSurface,
+       _onInverseSurface = onInverseSurface,
+       _inversePrimary = inversePrimary,
+       _primaryVariant = primaryVariant,
+       _secondaryVariant = secondaryVariant;
+
+  /// Generate a [ColorScheme] based around the given `seedColor`.
+  ///
+  /// Using the seedColor as a starting point, a set of tonal palettes are
+  /// constructed. These tonal palettes are based on the Material 3 Color
+  /// system and provide all the needed colors for a [ColorScheme]. These
+  /// colors are designed to work well together and meet contrast
+  /// requirements for accessibility.
+  ///
+  /// If any of the optional color parameters are non-null they will be
+  /// used in place of the generated colors for that field in the resulting
+  /// color scheme.
+  ///
```

Comment:
Hopefully a fun interactive color-scheme generator example appears here...

Reply:
It is on my list, but it would be a lot easier if we had a built in color picker... 😄

Reply:
You could always use 3 or 4 sliders for a rough and ready color picker.  Adding an example in a separate PR would be fine.